examples/analog_hamiltonian_simulation/09_Noisy_quantum_evolution_of_Bell_states.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.        h = Pattern([np.random.rand() for _ in filling])
</pre><p>Why don't we use normally distributed random variables here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>It is because the value for pattern can only take values between 0 and 1. </p>